forge/routes/api/project.js

```diff
@@ -835,14 +835,25 @@ module.exports = async function (app) {
             enabled: app.config.assistant?.enabled || false,
             requestTimeout: app.config.assistant?.requestTimeout || 60000,
             mcp: { enabled: true }, // default to enabled
-            completions: { enabled: true } // default to enabled
+            completions: { enabled: true, inlineEnabled: false } // default to enabled
         }
         if (app.config.assistant?.mcp && typeof app.config.assistant.mcp === 'object') {
             settings.assistant.mcp = { ...app.config.assistant.mcp }
         }
         if (app.config.assistant?.completions && typeof app.config.assistant.completions === 'object') {
             settings.assistant.completions = { ...app.config.assistant.completions }
         }
+        const isLicensed = app.license.active() || false
+        const licenseType = isLicensed ? (app.license.get('dev') ? 'DEV' : 'EE') : 'CE'
+        const tier = isLicensed ? app.license.get('tier') : null
+        const completionsTiers = [
+            'starter',
+            'teams', // AKA "pro" tier
+            'enterprise'
+        ]
```

Comment:
This is also very brittle, what happens when we intoduce a new team type next week (given the churn in billing this is VERY likely)

Reply:
Glad you picked up on that. I was unsure how we could achieve this any other way so this MVP hardcodes it.

For reference, the issue request is that "pro" (teams tier) and above are permitted to use this.

IIRC, we have no means of understanding what tiers are available (or, to be more precise, what _price_ordering_ they are)

Any thoughts?

Reply:
Added feature flag in [Enable Inline Completions via feature flag](https://github.com/FlowFuse/flowfuse/pull/5979/commits/c553e2bdf6abd77d421bf5603b0044f3e041eeae)